build-systems/github/snap.qt6/snapcraft.yaml

```diff
@@ -1,6 +1,5 @@
 name: qownnotes
 version: git
-version-script: cat src/version.h | sed "s/[^0-9,.]//g"
```

Comment:
The version needs to be set somehow...

Reply:
Yes. But not sure how. I would suggest reusing the qt5 snap script, if that works then porting it to Qt6 should be trivial

The snap changes in this MR are mostly experiments, feel free to take over